chronos.cabal

```diff
@@ -57,10 +57,11 @@ library
     -- it is OPTIONS_HADDOCK-hidden
       Chronos.Internal.CTimespec
   build-depends:
-    , aeson >= 1.1 && < 1.5
+      aeson >= 1.1 && < 1.5
```

Comment:
Can you put these leading commas back? They help me sort dependencies lists in my text editor.

Reply:
oh sorry about that. My editor did some funny reformatting on the cabal file and i missed that part.